Mailbox/Person.cs

```diff
@@ -1,4 +1,62 @@
-﻿namespace Mailbox
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+    public struct Person : IEquatable<Person> 
+    {
+        private string FirstName { get; set; }
+        private string LastName { get; set; }
```

Comment:
Properties can be made readonly
```suggestion
        private string LastName { get; }
```